src/radio/Radio.tsx

```diff
@@ -45,7 +45,7 @@ export type RadioProps = {
   /**
    * The actual text to be displayed `Cats For Adoptions`
    */
-  label: string;
+  label: string | ReactNode;
```

Comment:
I'm okay with this but note that this has a bit of a11y implications. Can you double check that the text node of whatever react node you pass in get's read in for a11y?